distributor-widget/README.md

```diff
@@ -1,41 +1,71 @@
+* Optional: Know the difference between [single and multi-enterprise Distributor](./README.md#multi-enterprise-distributor) and set it up.
+
+## Install Distributor loader script
+
+To use Distributor Widget, you need to install Distributor loader script with a code snippet provided in the [Installation](./README.md#installation) section.
+
+The script will asynchronously prepare global `Mews.Distributor` object which you're going to use in further steps to initialize Distributor Widget.
 
-The script should be included in the`<head>`section \(do not attach the`async`attribute\). The script size is kept as minimal as possible \(cca 11 kB gzipped\) to allow quick webpage initialization. The widget script is then downloaded asynchronously and inserted automatically into your website afterwards.
+### Requirements
 
-Please note that serving the script from our CDN servers ensures seamless releases of new features, bugfixes and improvements. Therefore, we discourage you from packing the contents of this script into your own JavaScript bundle. Make sure to follow the recommended way of including the scripts via`<script>`HTML tag.
+You need to use the code snippet as is and as described. Doing otherwise will cause unexpected problems and is not supported.
+
+* Do not place it anywhere else than in the `<head>`.
+* Do not modify it in any way and do not attach the `async` attribute.
+* Do not pack the contents of the script files that the code snippet references into your own JavaScript bundle.
+* If you have a Content Security Policy (CSP) setup on your website, you need to [enable the domains Distributor uses](./README.md#content-security-policy).
+
+The script file size is kept as minimal as possible (approx 11 kB gzipped) to allow quick webpage initialization. Also, serving the script from our CDN servers ensures seamless releases of new features, bugfixes and improvements.
+
+### Installation
+
+Place the following `<script>` code snippet as is in the `<head>` of your web page's HTML output, preferably as close to the opening `<head>` tag as possible.
+
+```html
+<script src="https://www.mews.li/distributor/distributor.min.js"></script>
+```
 
 #### Content Security Policy
 
-If you have a CSP setup on your website, the following domains should be enabled for distributor to function correctly.
+If you have a Content Security Policy (CSP) setup on your website, the following domains should be enabled for Distributor to function correctly.
 
 ```text
 mews.li
 *.mews.li
 https://pay.datatrans.com/upp/payment/js/secure-fields-1.0.0.js
 ```
 
-The final datatrans url is for [PCI Proxy](https://www.pci-proxy.com/) which is the secure, PCI-DSS complaint solution that is used by our Merchant to process payment cards.
+The last, datatrans, URL is for [PCI Proxy](https://www.pci-proxy.com/) which is the secure, PCI-DSS complaint solution that is used by our Merchant to process payment cards.
+
+## Initialize Distributor Widget
```

Comment:
I forgot if `Distributor Widget` should be both uppercase (it's a special name) or not (generic name). Same for Distributor Standalone. What do you think? But let's not fix it in this PR, too keep it consistent with other sections of the guide and a changes a little smaller.